apps/blur/test.cpp

```diff
@@ -49,9 +49,9 @@ Image<uint16_t> blur_fast(Image<uint16_t> in) {
                 for (int y = 0; y < 32+2; y++) {
                     const uint16_t *inPtr = &(in(xTile, yTile+y));
                     for (int x = 0; x < 128; x += 8) {
-                        a = _mm_load_si128((__m128i*)(inPtr));
-                        b = _mm_loadu_si128((__m128i*)(inPtr+1));
-                        c = _mm_loadu_si128((__m128i*)(inPtr+2));
+                        a = _mm_load_si128((const __m128i*)(inPtr));
```

Comment:
Bazel compilation with default C++ settings on OSX generated "constness cast away" warnings; fixing warnings is generally preferable to disabling them.